### PR TITLE
Added Kaggle kernels section

### PR DESCRIPTION
**Pull request by init27**
Added kaggle kernels link, getting started tutorial and returning to work section. 

I've also updated lesson notes with the kaggle kernel links as well as lesson 5,6,7 with resources.

### CODE REVIEW

File: files/dl-2019/notes/notes-1-6.md [hunk 1]
```diff
@@ -14,10 +14,34 @@ We'll use this knowledge to create a *class activated map*, which is a heat-map
...
+  - [rossman_data_clean](https://www.kaggle.com/init27/fastai-v3-rossman-data-clean)
+  - [lesson6-pets-more](https://www.kaggle.com/init27/fastai-v3-lesson-6-pets)
+- [Lesson 6 in-class discussion thread](https://forums.fast.ai/t/lesson-6-in-class-discussion/31440)
+- [Lesson 6 advanced discussion](https://forums.fast.ai/t/lesson-6-advanced-discussion/31442)
+
+### Other Resources
 
 - [platform.ai discussion](https://forums.fast.ai/t/platform-ai-discussion/31445)
 - [50 Years of Test (Un)fairness: Lessons for Machine Learning](https://128.84.21.199/pdf/1811.10104.pdf)
+- Convolutions: http://www.cs.cornell.edu/courses/cs1114/2013sp/sections/S06_convolution.pdf
```
Comment (jph00) [inline files/dl-2019/notes/notes-1-6.md]: These are not correctly formatted links.

File: docs/_data/sidebars/home_sidebar.yml [hunk 2]
```diff
@@ -39,6 +39,9 @@ entries:
   - title: Server setup
     output: web
     folderitems:
+    - output: web
```
Comment (jph00) [inline docs/_data/sidebars/home_sidebar.yml]: Move it to be last in the 'easy' section

File: docs/index.md [hunk 3]
```diff
@@ -28,6 +28,7 @@ Here are some great choices of platforms. Click the link for more information on
 
 These are the easiest to use; they've got all the software, data, and lessons preinstalled for you. They're a little less flexible than "full servers" (below), but are the simplest way to get started.
 
+- [Kaggle Kernels](/start_kaggle.html); (Instant Launch, No setup required, Free)
```
Comment (jph00) [inline docs/index.md]: Move this after 'colab'

File: docs/start_kaggle.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
+---
+
+title: Crestle.ai
```
Comment (jph00) [inline docs/start_kaggle.md]: Wrong title

File: docs/start_kaggle.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+keywords: neuralnets deeplearning
+sidebar: home_sidebar
+
+---
+# Welcome to Kaggle kernels!
+<img alt="" src="/images/kaggle/landing_page.png" class="screenshot">
+
+[Kaggle](https://kaggle.com/) is an online community of data scientists and machine learners, owned by Google, Inc. Kaggle allows users to find and publish data sets, explore and build models in a web-based data-science environment, work with other data scientists and machine learning engineers, and enter competitions to solve data science challenges. Kaggle got its start by offering machine learning competitions and now also offers a public data platform, a cloud-based workbench for data science. ([Read more here](https://en.wikipedia.org/wiki/Kaggle)
```
Comment (jph00) [inline docs/start_kaggle.md]: Please describe the downsides of kaggle kernels too. E.g. not updated at same frequency as official notebooks; not officially supported; RAM limitations (?)

File: files/dl-2019/notes/notes-1-1.md [hunk 6]
```diff
@@ -22,9 +22,10 @@ If you want to more deeply understand how PyTorch really works, you may want to
 - [Course repo](https://github.com/fastai/course-v3)
 - [fastai docs](http://docs.fast.ai)
 - [fastai datasets](https://course.fast.ai/datasets)
-- Notebooks:
   - [00_notebook_tutorial.ipynb](https://nbviewer.jupyter.org/github/fastai/course-v3/blob/master/nbs/dl1/00_notebook_tutorial.ipynb)
   - [lesson1-pets.ipynb](https://github.com/fastai/course-v3/blob/master/nbs/dl1/lesson1-pets.ipynb)
+- Kaggle Kernels:
```
Comment (jph00) [inline files/dl-2019/notes/notes-1-1.md]: Let's not modify any of the notes files.

Comment (jph00) [reply files/dl-2019/notes/notes-1-1.md]: (Sorry  - I mean, let's not add the kaggle kernels to the notes files. Your other edits are appropriate however)

File: docs/start_kaggle.md (hunk 4 above)
Comment (init27) [reply docs/start_kaggle.md]: Corrected, sorry.

File: docs/start_kaggle.md (hunk 5 above)
Comment (init27) [reply docs/start_kaggle.md]: I've added a Resources & Limitations section

File: files/dl-2019/notes/notes-1-1.md (hunk 6 above)
Comment (init27) [reply files/dl-2019/notes/notes-1-1.md]: I've removed the kernel links from all the notes and fixed the links in lesson 5,6,7 notes.

**Comment by init27** [comment]
I've made the suggested changes, thanks.

**Pull request merged**